docs/advanced_usage/advanced_config_file.rst

```diff
@@ -131,6 +131,12 @@ that will have exclusive access to the communication channel. A
 the created :py:class:`~pykiso.lib.auxiliaries.proxy_auxiliary.ProxyAuxiliary`. This allows to
 keep a minimalistic :py:class:`~pykiso.connector.CChannel` implementation.
 
+Access to attributes and methods of the defined communication channel that has been
+attached to the created :py:class:`~pykiso.lib.auxiliaries.proxy_auxiliary.ProxyAuxiliary`
+is still possible, but keep in mind that if one auxiliary modifies one the communication
+channel's attributes, every other auxiliary sharing this communication channel will be
+affected by this change.
```

Comment:
I think we should illustrate that with a python snippet somewhere

Reply:
To me it is sufficient to have this behaviour explained with text and then illustrated with the schema linked below, adding a code snippet here will add a lot of content for a side-note the user shouldn't usually have to care about